app/controllers/api/v1/ontologies_controller.rb

```diff
@@ -22,4 +23,8 @@ def version
     finder = OntologyVersionFinder.new(params[:reference], resource)
     @version ||= finder.find if params[:reference]
   end
+
+  def set_format
```

Comment:
I agree with @ebolloff, but please add a comment in the code like in app/fake_records/proof.rb:109